homeassistant/components/pvpc_hourly_pricing/config_flow.py

```diff
@@ -77,13 +77,7 @@ async def async_step_user(
             self._power = user_input[ATTR_POWER]
             self._power_p3 = user_input[ATTR_POWER_P3]
             self._use_api_token = user_input[CONF_USE_API_TOKEN]
-            return self.async_show_form(
-                step_id="api_token",
-                data_schema=vol.Schema(
-                    {vol.Required(CONF_API_TOKEN, default=self._api_token): str}
-                ),
-                description_placeholders={"mail_to_link": _MAIL_TO_LINK},
-            )
+            return await self.async_step_api_token(user_input)
```

Comment:
The user input passed here doesn't seem relevant or match the input expected in the api token step. I'd just call the method without passing user input. Otherwise, please explain what I'm missing

Reply:
Changed 👌